app/controllers/api/databases.php

```diff
@@ -2031,70 +2040,43 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
             throw new Exception(Exception::USER_UNAUTHORIZED);
         }
 
-        if (!empty($queries)) {
-            $attributes = array_merge(
-                $collection->getAttribute('attributes', []),
-                [
-                    new Document([
-                        'key' => '$id',
-                        'type' => Database::VAR_STRING,
-                        'array' => false,
-                    ]),
-                    new Document([
-                        'key' => '$createdAt',
-                        'type' => Database::VAR_DATETIME,
-                        'array' => false,
-                    ]),
-                    new Document([
-                        'key' => '$updatedAt',
-                        'type' => Database::VAR_DATETIME,
-                        'array' => false,
-                    ]),
-                ]
-            );
-            $validator = new IndexedQueries(
-                $attributes,
-                $collection->getAttribute('indexes', []),
-                new CursorQueryValidator(),
-                new FilterQueryValidator($attributes),
-                new LimitQueryValidator(),
-                new OffsetQueryValidator(),
-                new OrderQueryValidator(),
-            );
-            if (!$validator->isValid($queries)) {
-                throw new Exception(Exception::GENERAL_QUERY_INVALID, $validator->getDescription());
-            }
+        // Validate queries
+        $validator = new Documents($collection->getAttribute('attributes'), $collection->getAttribute('indexes'));
+        $valid = $validator->isValid($collection->getRead());
```

Comment:
Should this be `$queries` rather than `$collection->getRead()`?

Reply:
What a catch! That fixed a few tests, thanks! 😅